game/theater/controlpoint.py

```diff
@@ -1011,8 +1064,14 @@ def runway_is_operational(self) -> bool:
                 if db.unit_type_from_name(u.type) in [
                     Forrestal,
                     Stennis,
+                    VINSON,
```

Comment:
This list shouldn't be copy pasted everywhere. Factor out `game.db.CARRIERS` and `game.db.LHAS`. It seems like you're also repeating what is effectively `NavalControlPoint.get_carrier_unit` a lot.

Reply:
Done in 2352fbfcfef7ea43625b76fae0913733e326ed2a, thanks for the suggestion! Cleaned up the code quite a bit.